src/components/Lend/LoanSearch/ChallengeCallout.vue

```diff
@@ -1,15 +1,68 @@
 <template>
-	<div v-if="lenderName">
+	<div v-if="showAddedToCartMessage">
+		<kv-page-container class="container">
+			<kv-grid class="tw-grid-cols-12">
+				<div class="tw-col-span-12 tw-w-full">
+					<div class="info tw-w-full">
+						<div class="tw-flex tw-gap-1 tw-items-center">
+							<div class="tw-shrink-0">
+								<img
```

Comment:
Should we use KvUserAvatar here?
Looks like hiding users with no image...

Reply:
True, I was just re-using code from another component. Changed to avatar component.